securedrop/journalist_app/admin.py

```diff
@@ -52,8 +52,8 @@ def manage_config() -> Union[str, werkzeug.Response]:
                 f.save(custom_logo_filepath)
                 flash(gettext("Image updated."), "logo-success")
             except Exception:
-                flash("Unable to process the image file."
-                      " Try another one.", "logo-error")
+                # Translators: This error is shown when an uploaded image cannot be used.
```

Comment:
I think this is a good comment for devs as well as translators so perhaps remove `Translators:`

Reply:
The `Translators:` prefix is necessary for the comment to be carried over to the `.po` files, so it will appear in the Weblate UI.